tspng/extraction.py

```diff
@@ -144,3 +148,25 @@ def extract_from_folder(path: str, mime_type: str=MIME_TYPE) -> Dict:
     if file_list == []:
         raise Exception(f"The {path} does not contain a PNG file.")
     return extract_from_files(file_list, mime_type)
+
+def extract_from_url(url: str, mime_type: str=MIME_TYPE) -> Dict:
+    '''
+    Returns the metadata from a TS byte stream as a dictionary.
```

Comment:
Is this a "TS byte stream" or something more specific to a URL?

Reply:
It is not a byte stream, I will change it